app/src/main/java/org/astraea/app/performance/TrackerThread.java

```diff
@@ -175,6 +175,9 @@ static TrackerThread create(
               // if producers are not DONE, consumers should keep running as there are more data in
               // the future.
               if (!producerDone) consumerDone = false;
+              // if consumers are not DONE, they should keep consuming records when producers were
+              // Done.
+              if (producerDone && !consumerDone) consumerDone = logConsumers.apply(duration);
```

Comment:
``java
              if (!producerDone) producerDone = logProducers.apply(duration);
              // if producers are not DONE, consumers should keep running as there are more data in
              // the future.
              // if consumers are not DONE, they should keep consuming records when producers were
              // Done.
              if (!producerDone || !consumerDone) consumerDone = logConsumers.apply(duration);
```
麻煩看看這樣是否有更簡化